xrpVendordep/src/main/native/include/frc/xrp/XRPReflectanceSensor.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <frc/AnalogInput.h>
+
+namespace frc {
+
+/**
+ * This class represents the reflectance sensor pair
+ * on the XRP robot.
+ */
+class XRPReflectanceSensor {
+ public:
+  XRPReflectanceSensor() {}
+
+  /**
+   * Return the reflectance value of the left sensor.
+   * Value ranges from 0.0 (white) to 1.0 (black)
+   */
+  double GetLeftReflectanceValue();
```

Comment:
If we have reflectance units in the unit lib, use them.

Reply:
Reflectance has no units (since it's the ratio of light leaving a target area relative to light hitting the target area)